### PR TITLE
Improve JSDocs by adding examples of usages

### PR DESCRIPTION
**Pull request by RuneRogue**
@ivpavici 
Updated: 

✅ src/utils/calldata/formatter
✅ src/utils/calldata/tuple

### CODE REVIEW

File: src/utils/calldata/tuple.ts [hunk 1]
```diff
@@ -69,6 +91,11 @@ function getClosureOffset(input: string, open: string, close: string): number {
   return Number.POSITIVE_INFINITY;
 }
 
+/**
+ * Extract and parse a Cairo 1 tuple into its components.
```
Comment (ivpavici) [inline src/utils/calldata/tuple.ts:95]: jsdocs are unnecessary on all functions that are not exported! so you can remove here and on other functions above

**Comment by RuneRogue** [comment]
@ivpavici I have updated it.

**Comment by ivpavici** [comment]
Hi! 
We will double check and potentially merge this PR into https://github.com/starknet-io/starknet.js/pull/1221 !
cc @lukasaric

**Review by ivpavici — APPROVED** [review]

**Comment by ivpavici** [comment]
ok, merged and @lukasaric will continue in his PR if there are some changes to do

**Comment by ivpavici** [comment]
@RuneRogue sent you a small reward on OnlyDust!

**Comment by RuneRogue** [comment]
@ivpavici Thank you for reward.

**Pull request merged**